github/store/db.go

```diff
@@ -47,72 +47,180 @@ const (
+	}
+	_, err = s.DB.ExecContext(ctx, fmt.Sprintf(`CREATE MATERIALIZED VIEW issue_comments AS
+	SELECT c.repository_owner, c.repository_name, c.repository_owner || '/' || c.repository_name AS repository_full_name,
+	       c.issue_number, c.created_at, c.body, c.user_id, c.user_login, c.htmlurl AS html_url
+	FROM github_issue_comments_versioned AS c
+	JOIN github_issues_versioned AS i ON
+		 i.repository_owner = c.repository_owner AND
+		 i.repository_name = c.repository_name AND
+		 i.number = c.issue_number WHERE %v = ANY(c.versions)`, v))
+	if err != nil {
+		return fmt.Errorf("failed to create unified view issue_comments: %v", err)
+	}
+
+	_, err = s.DB.ExecContext(ctx, "DROP MATERIALIZED VIEW IF EXISTS pull_requests")
+	if err != nil {
+		return fmt.Errorf("failed to drop unified view pull_requests: %v", err)
+	}
+	_, err = s.DB.ExecContext(ctx, fmt.Sprintf(`CREATE MATERIALIZED VIEW pull_requests AS
+	SELECT repository_owner, repository_name, repository_owner || '/' || repository_name AS repository_full_name,
+	       number, state, title, body, created_at, closed_at, merged_at, updated_at,
+	       commits, comments, changed_files, additions, deletions, review_comments AS reviews,
+	       user_id, user_login, base_repository_name, base_repository_owner,
+	       base_repository_name || '/' || base_repository_owner AS base_repository_full_name,
+	       head_ref, head_sha, merge_commit_sha, htmlurl AS html_url, labels
+	FROM github_pull_requests_versioned WHERE %v = ANY(versions)`, v))
+	if err != nil {
+		return fmt.Errorf("failed to create unified view pull_requests: %v", err)
+	}
+
+	_, err = s.DB.ExecContext(ctx, "DROP MATERIALIZED VIEW IF EXISTS pull_request_reviews")
+	if err != nil {
+		return fmt.Errorf("failed to drop unified view pull_request_reviews: %v", err)
+	}
+	// TODO: state value rename COMMENTED to something else?
+	_, err = s.DB.ExecContext(ctx, fmt.Sprintf(`CREATE MATERIALIZED VIEW pull_request_reviews AS
+	SELECT repository_owner, repository_name, repository_owner || '/' || repository_name AS repository_full_name,
+	       pull_request_number, state, submitted_at AS created_at, user_id, user_login, htmlurl AS html_url
+	FROM github_pull_request_reviews_versioned WHERE %v = ANY(versions)`, v))
+	if err != nil {
+		return fmt.Errorf("failed to create VIEW pull_request_reviews: %v", err)
+	}
+
+	_, err = s.DB.ExecContext(ctx, "DROP MATERIALIZED VIEW IF EXISTS pull_request_comments")
+	if err != nil {
+		return fmt.Errorf("failed to drop unified view pull_request_comments: %v", err)
+	}
+	_, err = s.DB.ExecContext(ctx, fmt.Sprintf(`CREATE MATERIALIZED VIEW pull_request_comments AS
+	SELECT c.repository_owner, c.repository_name, c.repository_owner || '/' || c.repository_name AS repository_full_name,
+		   issue_number AS pull_request_number, c.created_at, c.body, c.user_id, c.user_login, c.htmlurl AS html_url
+	FROM github_issue_comments_versioned AS c
```

Comment:
why can't we use only the content of `github_pull_request_comments_versioned`?

Reply:
Note for myself: `UNION` is missing here, will need to fix.

Reply:
this was fixed